utilities/constants.py

```diff
@@ -1001,3 +1001,5 @@ class NamespacesNames:
 }
 
 ARQ_QUOTA_HARD_SPEC = {**QUOTA_FOR_POD, **QUOTA_FOR_ONE_VMI}
+
+EUS = "eus"
```

Comment:
just a nit: how about moving it above, for example under `# Upgrade tests configuration` group